web/packages/core/src/load-options.ts

```diff
@@ -117,6 +117,39 @@ export const enum WindowMode {
     Gpu = "gpu",
 }
 
+/**
+ * The render backend of a Ruffle player.
```

Comment:
I'd probably put something here to the effect of "these options are non exhaustive and may change at any point"

Reply:
Actually the RenderBackend enum exhaustively lists all of Ruffle's web render backends. Added the "may change at any point" part though.